src/styles/days.scss

```diff
@@ -0,0 +1,24 @@
+$indexes:
```

Comment:
Use correct file names and file structure.

More about file structure in scss in this [video, timing 1:00](https://mate.academy/learn/css-advanced/sass#/video/931)